copulas/multivariate/gaussian.py

```diff
@@ -185,19 +185,19 @@ def _get_conditional_distribution(self, conditions):
             tuple:
                 * means (numpy.array):
                     mean values to use for the conditioned multivariate normal.
-                * covariance (numpy.array):
-                    covariance matrix to use for the conditioned
+                * correlation (numpy.array):
```

Comment:
I think this method actually computes the covariance, since that's what the `multivariate_normal` needs.